### ISSUE TITLE
Color Scheme settings not working

### ISSUE THREAD

**Issue by avinizhanov** [noctule-ide/noctule-swift-plugin#25]
I'm not sure If I'm doing something wrong, but changing color scheme setting not working. 

![Screenshot 2023-09-11 at 9 21 59 AM](https://github.com/jansorg/swift-plugin/assets/42622715/0dc5010d-ee9b-4232-9412-c17ccbf8b18f)

![Screenshot 2023-09-11 at 9 21 39 AM](https://github.com/jansorg/swift-plugin/assets/42622715/a2f3ad31-7191-47a3-9183-107625d0a9f6)

Plugin version: 0.18.0.135-232
IDE: PhpStorm 2023.2.1

**Comment by jansorg** [comment]
Some of them are not yet in the preview, but should be applied in the editor. Some rely on the Swift toolchain, so you need to set it up to get the more advanced highlighting.

**Comment by avinizhanov** [comment]
@jansorg Yeah, looks like it works. But is there a way to set separate color for struct name/method name/variable declaration? 

<img width="827" alt="Screenshot 2023-11-09 at 2 17 36 PM" src="https://github.com/jansorg/swift-plugin/assets/42622715/81ff7b3f-a63c-4aa4-8088-2ab8613c40df">

**Comment by jansorg** [comment]
The latest version 1.7 has improved highlighting, including for method names and declarations (as far as SourceKit LSP allows).
